### PR TITLE
Feature: Adding FAQs to Pricing Page

### PR DESCRIPTION
**Pull request by dericksozo**
…he main amplication website, and adding some questions that we'll also add to the in-app pricing page.

Fixes: [amplication/private-issues#65](https://github.com/orgs/amplication/projects/5/views/1?pane=issue&itemId=45414876)

## PR Checklist

- [ ] `npm run test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

**Review by PazYanoverr — APPROVED** [review]

**Comment by PazYanoverr** [comment]
Waiting with the merge for the reverse trial and new packaging to be implemented

File: components/Sections/Pricing/PricingFAQ/index.jsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            <Question
+              question="We have multiple teams in the organization, can we use one account to manage separate teams?"
+              answer="Yes, the Enterprise plan lets you have unlimited projects, services, and team members. You can manage all your teams from the same organization in your preferred structure."
+            />
+            <Question
+            question="Can I pay to Amplication through my AWS account (AWS billing)?"
+            answer="Yes, we support this billing method. Please contact us and we will help you handle it."
+            />
+            <Question
+              question="We have our own best practices and standards - can we ask Amplication to generate the code in our own flavor?"
```
Comment (PazYanoverr) [inline components/Sections/Pricing/PricingFAQ/index.jsx:61]: Let's talk here also about the private plugins

**Review by PazYanoverr — CHANGES_REQUESTED** [review]
Let's improve the visibility of the FAQ-
1. It's hard to distinguish between the questions and the answer, so let's try to improve it.
2. The text is in **bold** - no need for that

**Comment by dericksozo** [comment]
Hi @PazYanoverr, the above two commits implements your feedback. Please let me know if there are any additional changes to be made.

**Review by yuval-hazaz — APPROVED** [review]

**Pull request merged**